lib/igniter/project/application.ex

```diff
@@ -64,7 +64,7 @@ defmodule Igniter.Project.Application do
       end
 
     Igniter.update_elixir_file(igniter, path, fn zipper ->
-      with {:ok, zipper} <- Igniter.Code.Module.move_to_module_using(zipper, Application),
+      with {:ok, zipper} <- Igniter.Code.Module.move_to_module_using(zipper, application),
```

Comment:
What this should be doing is moving to a module that contains this

```elixir
use Application
```

With this change, it would be looking for something like:

```elixir
use MyApp.Application
```

Which would be incorrect